INIT.md

```diff
@@ -1,10 +1,45 @@
-## webpack init
-
-Through [yeoman](http://yeoman.io/), the `webpack-cli init` feature allows people to create scaffolds and generate new projects quickly. An npm dependency that scaffolds a `webpack.config.js` through `webpack-cli` is what we refer to as an **addon**.
-
-We ask several questions in the default generator to get you started.
-
----
+# webpack-cli init
+
+`webpack-cli init` is used to initialize `webpack` projects quickly by scaffolding configuration and installing modules required for the project as per user preferences.
+
+## Initial Setup
+Follow given steps to setup `webpack-cli init` by installing dependencies:
+1. Create `package.json` through npm
+
+	```shell
+    	$ npm init
+    ```
+
+2. Install `webpack` and `webpack-cli` as devDependencies
+
+	```shell
+    	$ npm install --save-dev webpack webpack-cli
+    ```
+
+3. Optional step : Install @webpack-cli/init package to add init addon
```

Comment:
This shouldn't be optional I guess because this document is exclusive for the init command.

Reply:
When webpack-cli init is runned it install the package on its own.
Given step 2 is followed

Reply:
I think it should mandatory, not optional